README.md

```diff
@@ -100,77 +91,19 @@ You may use `dump-sql` parameter for dump sql with `DROP/CRATE INDEX` commands
 php app/console intaro:doctrine:index:update --dump-sql
 ```
 
-## Some annotation examples
+## Examples
 
 Create index using `pg_trgm` extension:
 ```
```

Comment:
add php type of code to highlight attribute

Reply:
add